### DISCUSSION TITLE
Background Audio Mixing on iOS App

### DISCUSSION THREAD

**Discussion by Mark02420** [chuot/rdio-scanner#296] [Ideas]
Would it be possible to add to the iOS app a selectable option to allow mixing of scanner audio with other audio such as podcasts or music etc.?  Scanner apps such as Action Scanner Radio Pro and Scanner have this feature and I use it all the time. As a bonus, the ability to adjust the relative app audio level from 50-150% would add to the mixing functionality. The primary use case is to have the scanner audio playing in the background while listening to music or podcasts over CarPlay in the car.

**Comment by chuot** [comment]
Believe me, I've tried to get it to work that way, but I'm limited by the framework I'm using plus the restrictions imposed by Apple.
